### PR TITLE
Add @indoorequal/vue-maplibre-gl

### PR DESCRIPTION
**Pull request by francois2metz**
This is a fork of https://github.com/razorness/vue-maplibre-gl with added features and support for maplibre-gl 4

### CODE REVIEW

**Comment by HarelM** [comment]
Wouldn't it be easier to simply upstream the work?

**Comment by francois2metz** [comment]
I tried, but there is no answers on my PR since 10 months :(

**Comment by HarelM** [comment]
I've added a comment to the relevant PR.
The last commit was 4 month ago, so I hope this repo is still maintained.
Ping me in a few days here if there's no response in the other issue and we'll replace the link instead of adding a new one.

**Comment by jokroese** [comment]
To follow up on this, there's still been no response (or commits or anything). @francois2metz's fork is very active and adding new features.

File: README.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ A collection of awesome things that use or support [MapLibre](https://maplibre.o
 ### Vue
 
 - [vue-maplibre-gl](https://github.com/razorness/vue-maplibre-gl) - Vue 3 plugin for maplibre-gl-js
+- [@indoorequal/vue-maplibre-gl](https://github.com/indoorequal/vue-maplibre-gl) - Vue 3 plugin for maplibre-gl-js that support the version 4 and add other new features
```
Comment (HarelM) [inline README.md:56]: I would suggest to replace the link completely.

**Review by HarelM — APPROVED** [review]

**Pull request merged**